app/api/v2/identity/sessions.rb

```diff
@@ -20,6 +20,11 @@ class Sessions < Grape::API
                                         desc: 'Response from Recaptcha widget'
           optional :otp_code, type: String,
                               desc: 'Code from Google Authenticator'
+          optional :geetest_response, type: Hash do
```

Comment:
Here we can leave only optional `:captcha_response, type: any`
No need to specify exact type and requires, as we dont need the captcha type at this point